frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -75,33 +76,38 @@ export default function SideNavigation({
 }: SideNavigationProps) {
   const [reviewCount, setReviewCount] = useState(0)
   const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+  const { responses, isResponsesLoading, isResponsesError } = useGetResponses(reviews.map((review) => review._id))
 
   const theme = useTheme()
 
-  const doesNotContainUserResponse = useCallback(
-    (review: ReviewRequestInterface) => {
-      return currentUser && !review.responses.find((response) => response.user === `user:${currentUser.dn}`)
-    },
-    [currentUser],
-  )
-
   useEffect(() => {
     async function fetchReviewCount() {
       onResetErrorMessage()
       if (reviews) {
-        setReviewCount(reviews.filter((filteredReview) => doesNotContainUserResponse(filteredReview)).length)
+        setReviewCount(
+          reviews.filter(
+            (filteredReview) =>
```

Comment:
Super picky, but the review we're iterating over hasn't actually been filtered at this point. So `filteredReview` is a bit of a confusing name. Consider renaming to `review`